Ship_Game/GameScreens/ShipDesign/ShipDesignScreen.cs

```diff
@@ -640,5 +644,18 @@ void RefreshHullSelectList()
                 }
             }
         }
+
+        public void ResetMusic()

```

Comment:
Why is this public, if this is referenced only from the same file?

Reply:
sorted out